app/components/ui/atoms/CalloutTextBox.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const Frame = styled(Box)`
+  border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorPrimary')};
+  background-color: ${th('colorPrimary')}20;
```

Comment:
Am I allowed to do this? 😁

Reply:
Looks like you have... It does assume the format of 'colorPrimary' ... Maybe we should put a comment near its definition?

Reply:
I think it's safe to assume that all of our colours, coming from our theme are in hex format. But I agree that maybe we should put a comment above this line, clarifying that the added 20 relates to the opacity. I imagine that this won't happen in many places within our code, but if it does, perhaps we should think about creating a helper function so that this is more obvious to read

Reply:
Do you reckon this helps, @diversemix ?